### PR TITLE
Add support for aten::select.int and aten::stack

### LINKED ISSUE #110: Add support for aten::select.int
## aten::select.int

- **Function Schema**: select.int(Tensor(a) self, int dim, int index) -> Tensor(a)

- **Original PyTorch API**: implicit PyTorch tensor op

- **Relevant TensorRT Documentation**:

### LINKED ISSUE #115: Add support for aten::stack
## aten::stack

- **Function Schema**:  stack(Tensor[] tensors, int dim=0) -> Tensor

- **Original PyTorch API**: https://pytorch.org/docs/master/generated/torch.stack.html

- **Relevant TensorRT Documentation**:

### PR DESCRIPTION
**Pull request by abhi-iyer**
# Description

Adds support for aten::select.int and aten::stack. Added test cases for these.

Added unsqueezeDims tool to unsqueeze Tensor at specified dimension. 

(from @narendasan) Added unpadDims tool to squeeze Tensor (removes first redundant dimension).

Fixes #110 
Fixes #115 

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
Closes #110
Closes #115

File: tests/core/converters/test_select.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include <string>
+#include "gtest/gtest.h"
+#include "torch/csrc/jit/ir/irparser.h"
+#include "tests/util/util.h"
+#include "core/compiler.h"
+
+TEST(Converters, ATenSelectIntTwiceConvertsCorrectly) {
```
Comment (narendasan) [inline tests/core/converters/test_select.cpp:7]: Can we add the single select test case as well?

Comment (abhi-iyer) [reply tests/core/converters/test_select.cpp:7]: sure
